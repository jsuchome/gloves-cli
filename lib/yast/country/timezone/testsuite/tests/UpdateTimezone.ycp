/**
 * tests for Timezone::UpdateTimezone
 * Author: jsuchome@suse.cz
 * $Id: UpdateTimezone.ycp 27936 2006-02-13 20:01:14Z olh $
 */
{

    // testedfiles: Timezone.ycp

    include "testsuite.ycp";

    map READ = $[
	"sysconfig" : $[
	    "clock" : $[
		"TIMEZONE"	: "Europe/Prague",
		"HWCLOCK"	: "-u",
	    ],
	],
	"target" : $[
	    "size"		: 1,
	    "yast2"		: $[],
	],
    ];
    map EXECUTE	= $[
	"target" : $[
	    "bash_output"	: $[],
	],
    ];

    TESTSUITE_INIT([READ,$[],EXECUTE], nil);

    import "Timezone";

    TEST (``(Timezone::UpdateTimezone ("US/Pacific")), [], []);

    TEST (``(Timezone::UpdateTimezone ("Australia/Adelaide")), [], []);

    TEST (``(Timezone::UpdateTimezone ("Australia/South")), [], []);
}
