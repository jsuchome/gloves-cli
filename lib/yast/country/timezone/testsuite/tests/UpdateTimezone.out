Return	America/Los_Angeles
Return	Australia/Adelaide
Return	Australia/Adelaide
